frontend/src/pages/instance/Assets.vue

```diff
@@ -88,6 +90,19 @@ export default {
         }
     },
     watch: {
+        teamMembership: {
+            handler (newState) {
+                if (newState && !this.hasAMinimumTeamRoleOf('member')) {
```

Comment:
Can we use the enum instead of hardcoded string?

e.g. `RoleNames[Roles.member]`

Better still if `hasAMinimumTeamRoleOf` accepted the actual role enum value e.g. `hasAMinimumTeamRoleOf(Roles.member)`

This way, any refactoring will be caught and references can be found.